cmake_modules/IcebergThirdpartyToolchain.cmake

```diff
@@ -65,8 +65,9 @@ function(resolve_arrow_dependency)
   set(ARROW_BUILD_STATIC
       ON
       CACHE BOOL "" FORCE)
+  # Workaround undefined symbol: arrow::ipc::ReadSchema(arrow::io::InputStream*, arrow::ipc::DictionaryMemo*)
```

Comment:
Ah, parquet-cpp can embed Arrow schemas in the Parquet metadata, so we need to be able to read IPC schemas. Possibly upstream should enable/require ARROW_IPC when Parquet is being built.